cypress/integration/home.spec.ts

```diff
@@ -0,0 +1,6 @@
+describe('home page', () => {
+  it('the p tag contains the correct text', () => {
```

Comment:
I think we can make this read a bit better

Since its a UI test, it would make more sense to describe the behavior of something rather than describe some technical aspect of what's being tested.

```suggestion
  it('displays the correct text', () => {
```

Reply:
Fixed on this [commit](https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/30/commits/4006342fb8cd9bc627cb3a1cc80b9fb474d110b5)